### ISSUE TITLE
Is this line still accurate?

### ISSUE THREAD

**Issue by bumblefudge** [decentralized-identity/waci-presentation-exchange#148]
> This version of the specification also restricts itself to [Verifiable Credentials](https://www.w3.org/TR/vc-data-model/) that make use of [BBS+ LD-Signatures](https://w3c-ccg.github.io/ldp-bbs2020).

I assume so, but:
 - I am unsure of the referent of "this version" - v0.1? v1?
 - perhaps I should go through and change every `specification` to `profile`, where the referent is this work item?

**Comment by OR13** [comment]
No, that line is inaccurate, we should refine this proposal to be JsonWebSignature2020 or Ed25519Siganture2018.
